### PR TITLE
feat(admin): user invitation form

### PR DESCRIPTION
**Pull request by jeysal**

### CODE REVIEW

File: apps/frontend/src/config.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+let baseUrl: string;
+switch (window.location.host.split('.').length) {
```
Comment (jeysal) [inline apps/frontend/src/config.ts:2]: This is ... less than ideal, ideas @fampinheiro?
I also don't really wanna pass the base URL via the environment when building though, that'd mean the same commit builds a different frontend bundle on `master` vs a branch, which is also weird

Comment (fampinheiro) [reply apps/frontend/src/config.ts:2]: Apart from an environment variable I'm not seeing many options.

File: serverless.js [hunk 2]
```diff
@@ -82,6 +82,9 @@ module.exports = {
     webpack: {
       config: 'serverless/webpack.config.js',
     },
+    'serverless-offline': {
+      useChildProcesses: true, // needed for hot reloading to work https://github.com/dherault/serverless-offline/issues/931
```
Comment (fampinheiro) [inline serverless.js:86]: Nice one.

**Comment by jeysal** [comment]
Two issues left before this is good to go:
* Two test TODOs left, will take care of them on Monday @jeysal 
* API requests fail due to missing CORS headers @fampinheiro

**Comment by jeysal** [comment]
removing any CORS experiments from this PR, can land those separately to make it *actually* work

**Pull request merged**